### PR TITLE
Issues/259 1

### PR DESCRIPTION
**Pull request by kholdaway**

### CODE REVIEW

File: quipucords/api/scanjob/serializer.py [hunk 1]
```diff
@@ -10,38 +10,39 @@
...
-        if isinstance(instance, Source):
-            display = 'Source: %s' % instance.name
-        return display
+    max_concurrency = IntegerField(required=False, min_value=1, default=50)
+
+    class Meta:
+        """Metadata for serializer."""
+
+        model = ScanOptions
+        fields = '__all__'
```
Comment (chambridge) [inline quipucords/api/scanjob/serializer.py]: should probably limit this to just the options fields like max_concurrency.

File: quipucords/api/scanjob/tests_scanjob.py [hunk 2]
```diff
@@ -51,40 +51,48 @@ def create(self, data):
...
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        response_json = response.json()
+        self.assertEqual(response_json, expected_response)
 
     def create_expect_201(self, data):
         """Create a source, return the response as a dict."""
         response = self.create(data)
+        response_json = response.json()
+        if response.status_code != status.HTTP_201_CREATED:
+            print('Cause of failure: ')
```
Comment (chambridge) [inline quipucords/api/scanjob/tests_scanjob.py]: Is this just for debugging?

File: quipucords/scanner/job.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        # Load tasks that have no been run or are in progress
+        tasks_to_run = []
+        incomplete_scantasks = self.scanjob.tasks.filter(
+            Q(status=ScanTask.RUNNING) | Q(status=ScanTask.PENDING)
+        ).order_by('sequence_number')
+        for scantaskmodel in incomplete_scantasks:
+            tasks_to_run.append(ScanTaskRunner(self.scanjob, scantaskmodel))
+
+        print('Running: %s' % self.scanjob)
```
Comment (chambridge) [inline quipucords/scanner/job.py]: maybe log instead of print.

File: quipucords/api/scanjob/tests_scanjob.py (hunk 2 above)
Comment (kholdaway) [reply quipucords/api/scanjob/tests_scanjob.py]: Yes.  When test failed unexpectedly (not a 201), print the reason. Helpful for debugging.

**Pull request merged**